package.json

```diff
@@ -130,6 +131,7 @@
     "meteor-promise": "0.8.7",
     "mock-req": "0.2.0",
     "multer": "1.4.1",
+    "nock": "^10.0.6",
```

Comment:
It looks like the norm in this repo is to add strict dependencies (which I believe are updated by a bot). I can fix that in #2900 if you'd like (which was also going to add nock, amusingly).

Reply:
Oh my bad, go ahead